### ISSUE TITLE
Post-training

### ISSUE THREAD

**Issue by Daturl** [anilbatra2185/road_connectivity#36]
Thank you for your great work. Is the post-training file released yet? Or please tell me the input of fine-tuning is the output of the test set in the mul network？thanks
